src/content/docs/synthetics/synthetic-monitoring/getting-started/get-started-synthetic-monitoring.mdx

```diff
@@ -16,32 +16,211 @@ redirects:
-* Expand your monitoring further with real, Selenium-powered [scripted browsers](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-synthetic-scripts), which test login procedures, searches, and other critical business transactions.
-* Monitor your API endpoints with [API tests](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-api-tests).
+Our synthetic monitors are more than just ping monitors. Synthetics can perform scriptable checks on your customer-facing or internal apps, down to the HTTP headers. By mimicking the behaviors of customers, your monitors can test how well your web apps can handle traffic. This lets you proactively catch and resolve issues, whether it's a broken link, a failed API call, or significant changes in your app runtime.
 
-Ready to get started? If you don't already have one, [sign up for a New Relic account](https://newrelic.com/signup). It's free, forever!
+Setting up synthetic monitors with New Relic lets you:
 
-Synthetic monitoring has countless use cases, and knowing the right monitor type for yours is important. To learn about our different types of synthetic monitors and the differences between each, watch this short video (5:10 minutes).
+* Expand your browser monitoring with real, Selenium-powered [scripted browsers](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-synthetic-scripts), which test login procedures, searches, and other critical business transactions.
+* Run an [API test](/docs/synthetics/new-relic-synthetics/scripting-monitors/writing-api-tests) to check certificates or make sure any kind of HTTP requests are completed. 
+* Get extended use case coverage with [additional Node modules](/docs/synthetics/synthetic-monitoring/scripting-monitors/import-nodejs-modules) for your scripted API monitors. 
+* Easily diagnose whether an issue stems from the network or AWS location, a slow third party resource, or the health of your backend services or infrastructure. 
+
+For a short demo of how to create synthetic monitors with New Relic, watch the below video (5:10 minutes).
 
 <Video
   id="EN5FkIW8YWk"
   type="youtube"
 />
 
-## Advanced testing [#advanced]
+## Before you create your first synthetic monitor...
+
+Once you've [signed up for a New Relic account](https://newrelic.com/signup), you're ready to create your first synthetic monitor. Below, you can get a high-level overview of system requirements, how we protect data, and how you can segment monitoring capabilities by roles within an organization.     
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="compatibility"
+    title="Are there any system requirements with synthetic monitoring?"
+  >
+Synthetic monitoring does not require any software except a [supported browser](/docs/apm/new-relic-apm/getting-started/supported-browsers).
+
+<Callout variant="important">
+  To monitor a site behind your firewall, add the [synthetic monitoring public
+  minion IP
```

Comment:
I don't know that this matters but there is some weird formatting here with this link.

Reply:
LOL. Yes.... this does matter, yes... good catch